src/force/nep3_small_box.cuh

```diff
@@ -315,12 +340,14 @@ static __global__ void find_descriptor_small_box(
   const float* __restrict__ g_x12_angular,
   const float* __restrict__ g_y12_angular,
   const float* __restrict__ g_z12_angular,
+  const bool is_polarizability,
```

Comment:
There is no need to modify the functions for the "temperature" version because in this case `nep_model_type=3`.

Reply:
Right, didn't think of that :facepalm: